Phonebook.Frontend/src/app/shared/components/navigation/navigation.component.ts

```diff
@@ -18,6 +18,7 @@ import { Environment, EnvironmentInterface } from 'src/environments/EnvironmentI
 import { runtimeEnvironment } from 'src/environments/runtime-environment';
 import { HASH_LONG, HASH_SHORT, VERSION } from 'src/environments/version';
 
+
```

Comment:
remove new line

Reply:
You can also leave it in, but please check the next time committing that you only check in files that have meaningful changes.